Microsoft.Alm.Authentication/VstsAadAuthentication.cs

```diff
@@ -78,6 +81,8 @@ public sealed class VstsAadAuthentication : BaseVstsAuthentication, IVstsAadAuth
                    vstsAuthority)
         { }
 
+        public String LoginHint { get; set; }
```

Comment:
Since this is a c'tor argument, is there a valid reason to allow it to be mutable? Mutability is usually not what we want.